qa/rpc-tests/zkey_import_export.py

```diff
@@ -69,6 +70,12 @@ def z_send(from_node, from_addr, to_addr, amount):
             miner.generate(1)
             self.sync_all()
 
+        def z_getbalance(node, zaddr):
+            bal = node.z_getbalance(zaddr)
+            # Ignore fees for sake of comparison
+            round_balance = math.ceil(bal*100)/100
```

Comment:
Is this blocking?

Reply:
I ask because I tried to do this, but it's returning a weird non-standard fee amount. Normal fee is `0.0001`, this returns with  `0.00019999999999997797` deducted.